MANIFESTO.md

```diff
@@ -0,0 +1,38 @@
+# Manifesto
+As Hitta.se's development team, we believe in...
+
+1. **Testing matters**
+   - We exponentially increase the amount of rework when we try to find and fix bugs later in any lifecycle.
+   - That's why we test early and test often.
+   - We always dedicate time to expand our automated test suites by building both unit and end-to-end tests.
+2. **Quality above all**
+   - Quality takes priority over features and time.
+   - Quality has many facets, such as security, performance and even a great user experience.
+   - Code we write in the morning may be in production by the afternoon.
```

Comment:
Not sure this goes hand-in-hand with Quality, perhaps more related to "Automation is essential"?

Reply:
Yes correct!